include/unifex/any_unique.hpp

```diff
@@ -395,6 +395,21 @@ class _byval<CPOs...>::type
     , vtable_(other.vtable_) {}
 
   UNIFEX_ALWAYS_INLINE ~type() {
+    unsafe_deallocate();
+  }
+
+  type& operator=(type&& other) noexcept {
+    unsafe_deallocate();
+    impl_ = std::exchange(other.impl_, nullptr);
+    vtable_ = std::move(other.vtable_);
```

Comment:
If we implemented `swap()` for `any_unique` then this could just be:
```
type& operator=(type t) noexcept {
  swap(t);
  return *this;
}
```
and would also then be self-assignment safe.